scripts/build-shaded-dtest-jar-local.sh

```diff
@@ -25,33 +25,33 @@ SCRIPT_DIR=$( dirname -- "$( readlink -f -- "$0"; )"; )
 CASSANDRA_VERSION=$(cat build.xml | grep 'property name="base.version"' | awk -F "\"" '{print $4}')
 GIT_HASH=$(git rev-parse --short HEAD)
 DTEST_ARTIFACT_ID=${ARTIFACT_NAME}-local
-DTEST_JAR_DIR="$(dirname ${SCRIPT_DIR}/)/dtest-jars"
+DTEST_JAR_DIR="$(dirname "${SCRIPT_DIR}/")/dtest-jars"
 
-echo $CASSANDRA_VERSION
-echo $GIT_HASH
-echo $DTEST_ARTIFACT_ID
+echo "${CASSANDRA_VERSION}"
+echo "${GIT_HASH}"
+echo "${DTEST_ARTIFACT_ID}"
 
 ant clean
 ant dtest-jar -Dno-checkstyle=true
 
 # Install the version that will be shaded
-mvn install:install-file               \
-   -Dfile=./build/dtest-${CASSANDRA_VERSION}.jar \
-   -DgroupId=org.apache.cassandra      \
-   -DartifactId=${DTEST_ARTIFACT_ID} \
-   -Dversion=${CASSANDRA_VERSION}-${GIT_HASH}         \
-   -Dpackaging=jar                     \
-   -DgeneratePom=true                  \
-   -DlocalRepositoryPath=${REPO_DIR}
+"${SCRIPT_DIR}/mvnw" install:install-file                            \
```

Comment:
should we consider windows development and run `mvnw.cmd`, or assume no such use case and discard the cmd file?

Reply:
didn't Cassandra drop support for windows? I would not support sidecar for windows if the parent project doesn't support it

Reply:
good call. Then we should remove the cmd file. And there is gradle.bat file could be removed too.

Reply:
done